packages/rs-dpp/src/identity/state_transition/identity_credit_withdrawal_transition/validation/basic/validate_identity_credit_withdrawal_transition_basic.rs

```diff
@@ -73,12 +73,14 @@ impl IdentityCreditWithdrawalTransitionBasicValidator {
 
         // validate core_fee is in fibonacci sequence
         let core_fee = transition_json
-            .get("coreFee")
+            .get("coreFeePerByte")
```

Comment:
Don't add the string here, instead use something like paramName::CoreFeePerByte. I think we have that construction on some other places in dpp.